lib/indexer/workers/amend_worker.rb

```diff
@@ -6,6 +6,7 @@ def perform(index_name, document_link, updates)
       logger.info "Amending document '#{document_link}' in '#{index_name}'"
       logger.info "Amending fields #{updates.keys.join(', ')}"
       logger.debug "Amendments: #{updates}"
+      return if updates.empty?
```

Comment:
I guess this is an improvement, but ideally these jobs wouldn't be enqueued in the first place. Do you know where they might be coming from?